data/topic_filter.go

```diff
@@ -15,7 +15,7 @@ import (
 // Topic represents a topic filter on the search page
 type Topic struct {
 	Count              int        `json:"count"`
-	DistinctTopicCount int        `json:"distinct_topics_count"`
```

Comment:
Maybe we should rename this variable to 'DistinctItemsCount' as well? (or would it cause too many issues?)